2022/incoming/create-your-excel-using-perl.pod

```diff
@@ -0,0 +1,170 @@
+Author: Csaba Simándi <x1m4nd1@gmail.com>
+Title: Create Your Excel using Perl!
+Topic: Excel::Writer::XLSX
+
+=encoding utf8
+
+=head1 Create Your Excel using Perl!
+
+=head2 Intro
+
+A long-long time ago, during the Advent of 2001, there was an article about manipulating Excel spreadsheets using Perl, L<Spreadsheet::WriteExcel|https://perladvent.org/2001/14th/>.
+It's been a while now, so Santa decided to ask the Wise Old Elf to write another article about creating Excel files using Perl. 
+
+There were only 3 days until Advent, with lots of articles missing, so the Wise Old Elf immediately delegated this task to Wunorse Openslae, and he chose wisely - as always. Wunorse was a master of managing Excel files and started to work on that article immediately.
+
+=head2 Preface
+
+Some historical background: The older module L<Spreadsheet::WriteExcel|https://metacpan.org/pod/Spreadsheet::WriteExcel> is for creating Excel files compatible with Excel 97, 2000, 2002, 2003 and 2007, mainly with C<.xls> extension. It is mostly compatible with Excel::Writer::XLSX, so L<migrating to the newer version|https://metacpan.org/pod/Spreadsheet::WriteExcel#Migrating-to-Excel::Writer::XLSX> should be as easy as Christmas pie. However such a migration is more like a heavy Christmas pudding loaded with goodness and lots of issues might arise during the process.
+
+You can use C<Spreasheet::ParseExcel> for reading the older Excel files having C<.xls> extension, on the other hand you should use C<Spreadsheet::ParseXLSX> or C<Spreadsheet::XLSX> to open and manipulate existing Excel files. This article deals with creating Excel files in C<.xlsx> format.
```

Comment:
This is indeed better.

One small change to make it even clearer - I presume you mean to use the newer modules on .xlsx files only so maybe add that?

eg: "on the other hand you should use C<Spreadsheet::ParseXLSX> or C<Spreadsheet::XLSX> to open and manipulate existing Excel files having the .xlsx extension"